OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -337,19 +337,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </extent>
                                     
                                     <collation>
+                                        <note>
+                                            All quires have been reinforced with parchment guards at a later stage. The original quire structure cannot be reconstructed anymore (see also Condition below).
+                                        </note>
                                         <signatures>
-                                            ፫ in the upper left corner of <locus target="#18r"/> and upper right corner of <locus target="#25v"/>;
-                                            ፬ in the upper left corner of <locus target="#26r"/>, remains of a quire mark in the upper right corner of <locus target="#31v"/>;
-                                            ፫ in the upper left corner of <locus target="#54r"/>, ፬ in the upper right corner of <locus target="#68v"/>;
-                                            ፪ n the upper left corner of  <locus target="#69r"/> and the upper right corner of <locus target="#76v"/>;
-                                            ፫ in the upper left corner of <locus target="#77r"/>;
-                                            remains of a quire mark in the upper left corner of <locus target="#85r"/>;
-                                            ፭ in the upper left corner of <locus target="#93r"/>;
-                                            ፩ in the upper right corner of <locus target="#108v"/>;
-                                            ፪ in the upper left corner of <locus target="#109r"/> and the upper right corner of <locus target="#115v"/>;
-                                            ፫ in the upper left corner of <locus target="#116r"/>.
-                                        <!-- These pieces of information can be summarized in one line --></signatures>
-                                        <note>All quires have been subsequently reinforced with parchment guards. The original quire structure cannot be reconstructed anymore.</note>
+                                            Quire marks were written in the upper inner margin of the first folio and in the upper inner margin of the last folio of most of the original quires,
+                                            which are now displaced. Therefore, the position of the signatures no longer coincides with the beginning and end of the current quires.    
```

Comment:
What does "displaced" mean here? That the quires are in the wrong order or that leaves were misplaced within individual quires?

Reply:
Fixed. The information was unnecessary. Thank you Jonas.